### PR TITLE
[PPP-4447] - Upgraded jackrabbit 2.16.5 to resolve issues with Tika v…

### PR DESCRIPTION
**Pull request by wseyler**
…ulnerabilities.  Also upgraded separate uses of tika.

### CODE REVIEW

**Comment by buildguy** [comment]
### :gear: Build finished in 37s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 35s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 31s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -147,6 +147,7 @@
     <commons-fileupload.version>1.4</commons-fileupload.version>
     <aws-java-sdk.version>1.11.516</aws-java-sdk.version>
     <jetty.version>8.1.15.v20140411</jetty.version>
+    <jackrabbit.version>2.16.5</jackrabbit.version>
```
Comment (pentaho-lmartins) [inline pom.xml]: Missing dependency management.

**Comment by buildguy** [comment]
### :gear: Build finished in 34s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

File: pom.xml (hunk 1 above)
Comment (wseyler) [reply pom.xml]: Done.

**Comment by buildguy** [comment]
### :x: Build failed in 9s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
Pull request 172 : Not mergeable at 5c22fab69fddb1af79579ccdafe68344521d59f6+92f48c55a492f29759e01d99b5166d4e198e59a9 (NOT_MERGEABLE)
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by pentaho-lmartins** [comment]
Closing in favor of #178.

**Pull request closed without merging**